### DISCUSSION TITLE
Add "Send to interactive window" in Jupyter Notebooks

### DISCUSSION THREAD

**Discussion by t-fourcade-isp** [microsoft/vscode-jupyter#10560] [New Ideas]
Hi,

When in jupyter notebooks, the command "send to interactive notebooks" is missing (not in right click in contextual menu nor in the palette)
 and shift+enter executes the current cell. The comportment is normal when working with plain python file.

<html>
<body>

CPUs | Intel(R) Xeon(R) W-1250P CPU @ 4.10GHz (12 x 4104)
-- | --
GPU Status | 2d_canvas: enabled canvas_oop_rasterization: disabled_off direct_rendering_display_compositor: disabled_off_ok gpu_compositing: enabled multiple_raster_threads: enabled_on oop_rasterization: enabled opengl: enabled_on rasterization: enabled raw_draw: disabled_off_ok skia_renderer: enabled_on video_decode: enabled video_encode: unavailable_off vulkan: disabled_off webgl: enabled webgl2: enabled
Load (avg) |  
Memory (System) | 31.78GB (23.38GB free)
Process Argv | --crash-reporter-id 8b83c450-0220-4526-b38e-6b90351fdd64
Screen Reader | no
VM | 0%

</body>
</html>

 
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vscaac:30438847
vsclangdc:30486549

**Comment by DonJayamanne** [comment]
Thanks for filing this issue, however this isn't a feature supported today.
Today when in Jupyter Notebooks, the expected behaviour is for users to run an individual cell. If you would like to  run an individual line then the expectation is to create a cell with just that one line.
The feature you are referring to is something that was added for Interactive window.

**Reply by t-fourcade-isp** [reply]
OK, my bad, It would be a nice feature thought even if we can easily live without it.

**Comment by DonJayamanne** [comment]
Trasferring this issue to general discussions as this is a feature request.
